Library/Formula/radare2.rb

```diff
@@ -38,24 +32,56 @@ class Radare2 < Formula
   depends_on "valabind" => :build
   depends_on "swig" => :build
   depends_on "gobject-introspection" => :build
+  depends_on "gmp"
   depends_on "libewf"
   depends_on "libmagic"
-  depends_on "gmp"
   depends_on "lua51" # It seems to latch onto Lua51 rather than Lua. Enquire this upstream.
   depends_on "openssl"
+  depends_on "yara"
 
   def install
     # Build Radare2 before bindings, otherwise compile = nope.
     system "./configure", "--prefix=#{prefix}", "--with-openssl"
-    system "make"
+    system "make", "CS_PATCHES=0"
```

Comment:
Out of curiosity, why the `CS_PATCHES=0`? Is this needed to keep it inside `prefix`, or for some other reason?